src/nvim/api/window.c

```diff
@@ -80,7 +80,7 @@ void nvim_win_set_buf(Window window, Buffer buffer, Error *err)
 ///
 /// @param window   Window handle, or 0 for current window
 /// @param[out] err Error details, if any
-/// @return (row, col) tuple
+/// @return (row, byte-offset) tuple
```

Comment:
I don't remember this to be the standardized terminology. Col=bytes after NL already, while vcol = screen colunm. If we change this we need to commit to change a lot of stuff

Reply:
Even in fact "byte-offset" is often the jargon in newer codes (like extmark tree sitter) for the total byte offset. So this would just be confusing anyway.